src/commands/compile.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
       default: true
     });
 
+    yargs.positional('ignore', {
+      describe: 'Glob pattern matching files that will not be updated',
```

Comment:
Should we add an additional note like: "It always ignores node_modules folders"

Reply:
Yes we should

Reply:
In fact, should we make it cancellable?
`Glob pattern matching files that will not be updated, by default node_modules are ignored but if you decide to use this option, you will need to add it to the ignore list` or something like that?

Reply:
I'll make an issue to discuss it :)